### PR TITLE
Use current platform for OpenJDK tests

### PR DESCRIPTION
**Pull request by jlahoda**
Without this, the OpenJDK tests do not see classes added in the OpenJDK sources, but rather the runtime platform classes (this is particularly problem for classes for java.base).

Erik, it would be awesome if this could be considered for 12.0.

### CODE REVIEW

File: java/refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/MoveMembersTransformer.java [hunk 1]
```diff
@@ -451,6 +460,9 @@ public Boolean reduce(Boolean r1, Boolean r2) {
                     return (r1 == Boolean.TRUE || r2 == Boolean.TRUE);
                 }
             };
+            if (workingCopy.getTrees().getTree(el) == null) {
+                System.err.println("!!!");
```
Comment (JaroslavTulach) [inline java/refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/MoveMembersTransformer.java:464]: Left over debug message, I guess.

File: java/refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/PushDownTransformer.java [hunk 2]
```diff
@@ -62,11 +67,18 @@ public Problem getProblem() {
...
+    public PushDownTransformer(FileObject originFile, MemberInfo<ElementHandle<? extends Element>> members[]) {
+        this.originFile = originFile;
         this.members = members;
         this.translateQueue = new LinkedList<>();
     }
 
+    @Override
+    public void setWorkingCopy(WorkingCopy workingCopy) throws ToPhaseException {
+        SourceUtils.forceSource(workingCopy, originFile);
+        super.setWorkingCopy(workingCopy); //To change body of generated methods, choose Tools | Templates.
```
Comment (JaroslavTulach) [inline java/refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/PushDownTransformer.java:79]: Delete the comment.

File: java/refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/MoveMembersTransformer.java (hunk 1 above)
Comment (jlahoda) [reply java/refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/MoveMembersTransformer.java]: Right. Sorry, I mistakenly based this branch on a different branch, fixed. There should only be one file changed, the ClassPathProvider.

File: java/refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/PushDownTransformer.java (hunk 2 above)
Comment (jlahoda) [reply java/refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/PushDownTransformer.java]: Right. Sorry, I mistakenly based this branch on a different branch, fixed. There should only be one file changed, the ClassPathProvider.

**Comment by arvindaprameya** [comment]
+1 Approved, Thanks

**Pull request merged**